beluga_ros/test/test_sparse_point_cloud.cpp

```diff
@@ -0,0 +1,225 @@
+// Copyright 2024 Ekumen, Inc.
```

Comment:
date

Reply:
Fixed in 4dd4e03aa2b1457381049d8d29f5c38ef18663d8.